### PR TITLE
`CHANGELOG.md` updates for `4.14.0` release

### PR DESCRIPTION
**Pull request by jackofallops**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 4.14.0 (Unreleased)
+
+ENHANCEMENTS:
+
+* `azurerm_linux_function_app` - add support for  preview  value`21` for `java_version` [GH-26304]
+* `azurerm_linux_web_app` - add support for  preview  value`21` for `java_version` [GH-26304]
+* `azurerm_windows_function_app` - add support for  preview  value`21` for `java_version` [GH-26304]
+* `azurerm_windows_web_app` - add support for  preview  value`21` for `java_version` [GH-26304]
```
Comment (WodansSon) [inline CHANGELOG.md:8]: Should there be a space here?
```suggestion
* `azurerm_linux_function_app` - add support for  preview  value `21` for `java_version` [GH-26304]
* `azurerm_linux_web_app` - add support for  preview  value `21` for `java_version` [GH-26304]
* `azurerm_windows_function_app` - add support for  preview  value `21` for `java_version` [GH-26304]
* `azurerm_windows_web_app` - add support for  preview  value `21` for `java_version` [GH-26304]
```

**Review by WodansSon — DISMISSED** [review]
This LGTM, but was there a space missed here? Very minor nit, but thought I would ask the question. 🚀

**Review by katbyte — DISMISSED** [review]
LGTM 🛋️

**Review by katbyte — APPROVED** [review]
LGTM test 🤔

**Review by stephybun — APPROVED** [review]
Third time's a charm?

**Pull request merged**